### ISSUE TITLE
Max in Flight: Document definition inaccurate

### ISSUE THREAD

**Issue by outofmem0ry** [pivotal-cf/docs-pks#355]
Current definition - Under Worker VM Max in Flight, enter the maximum number of worker VMs you want created.

This should be the maximum number of non-canary worker instances to update in parallel within an availability zone. Also, from the documentation it is not clear that if this parameter will take effect during upgrade of PKS clusters.

cc @kroberts7 

Reference: https://bosh.io/docs/manifest-v2/

**Comment by kdolor-pivotal** [comment]
@outofmem0ry Thanks for your feedback ! I have fixed the line.

We mention `max in flight` in upgrades here: https://github.com/pivotal-cf/docs-pks/blob/master/understanding-upgrades.html.md.erb

Is further explanation needed?

**Comment by outofmem0ry** [comment]
@kdolor-pivotal - there are few other issues with https://docs-pcf-staging.cfapps.io/runtimes/pks/1-2/understanding-upgrades.html#max-in-flight and we'll be creating a separate issue for that.

**Comment by kdolor-pivotal** [comment]
Thanks @outofmem0ry I will close this issue, as we await the forthcoming issue regarding https://docs-pcf-staging.cfapps.io/runtimes/pks/1-2/understanding-upgrades.html#max-in-flight
